bin/markergene_study_summary.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright 2024 EMBL - European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import argparse
+from collections import defaultdict
+import json
+import pathlib
+import logging
+
+import pandas as pd
+import pyfastx
+
+logging.basicConfig(level=logging.DEBUG)
+
+def parse_args():
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-i", "--input", required=True, type=str, help="Input directory containing amplicon analysis pipeline results")
+    parser.add_argument("-r", "--runs", required=True, type=str, help="CSV file containing successful analyses generated by the pipeline")
+    parser.add_argument("-p", "--prefix", required=True, type=str, help="Prefix for the output file")
+
+    args = parser.parse_args()
+
+    INPUT = args.input
+    RUNS = args.runs
+    PREFIX = args.prefix
+
+    return INPUT, RUNS, PREFIX
+
+def get_read_count(read_path):
+
+    fasta = pyfastx.Fasta(read_path, build_index=False)
+    read_count = sum(1 for _ in fasta)
+
+    return read_count
+
+def add_markergene(root_path, run_acc, markergene_dict, markergene):
+
+    if markergene != "ITS":
+
+        bacterial_ssu = list(pathlib.Path(f"{root_path}/{run_acc}/sequence-categorisation").glob(f"*{markergene}*bacteria*"))
+        archaeal_ssu = list(pathlib.Path(f"{root_path}/{run_acc}/sequence-categorisation").glob(f"*{markergene}*archaea*"))
+        eukarya_ssu = list(pathlib.Path(f"{root_path}/{run_acc}/sequence-categorisation").glob(f"*{markergene}*eukarya*"))
+
+        markergene_dict[f"{markergene} - bacteria"] = defaultdict()
```

Comment:
I think space is quite dangerous sign, even in json :D maybe replace it with underscore?